stablehlo/reference/Process.cpp

```diff
@@ -0,0 +1,20 @@
+/* Copyright 2023 The StableHLO Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "stablehlo/reference/Process.h"
+
+namespace mlir {
+namespace stablehlo {}  // namespace stablehlo
+}  // namespace mlir
```

Comment:
Do we need the .cpp file at this point given that it's pretty much empty right now?